package.json

```diff
@@ -21,8 +21,8 @@
   "scripts": {
     "prepublishOnly": "npm run build",
     "build": "tsc",
-    "test": "npm run build && mocha -t 10000 tests/*spec.js",
-    "snapshots": "percy exec -- npm run test"
+    "pretest": "npm run build",
+    "test": "percy exec -- mocha tests/*spec.js"
```

Comment:
FWIW, originally I had two separate commands (with and without snapshots) to make it possible to run the tests without having to start up the agent (which makes it possible to also test the SDK functionality with no running agent). But I guess at this point it is no longer necessary.

Reply:
yeah I think in this context this makes the most sense. There's a subtle behavior which is being tested here too... percy agent gracefully disabling itself when it's not in the right conditions to run.